pkg/agent/openflow/pipeline_windows.go

```diff
@@ -297,15 +108,30 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 
 func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, remoteGatewayMAC net.HardwareAddr,
 	category cookie.Category, peerIP net.IP, peerPodCIDR *net.IPNet) []binding.Flow {
-	if c.encapMode.NeedsDirectRoutingToPeer(peerIP, c.nodeConfig.NodeIPAddr) && remoteGatewayMAC != nil {
+	if c.encapMode.NeedsDirectRoutingToPeer(peerIP, c.nodeConfig.NodeTransportIPAddr) && remoteGatewayMAC != nil {
+		ipProto := getIPProtocol(peerIP)
+		l3FwdTable := c.pipeline[l3ForwardingTable]
 		// It enhances Windows Noencap mode performance by bypassing host network.
 		flows := []binding.Flow{c.pipeline[l2ForwardingCalcTable].BuildFlow(priorityNormal).
 			MatchDstMAC(remoteGatewayMAC).
 			Action().LoadRegRange(int(PortCacheReg), config.UplinkOFPort, ofPortRegRange).
 			Action().LoadRegRange(int(marksReg), macRewriteMark, ofPortMarkRange).
 			Action().GotoTable(conntrackCommitTable).
 			Cookie(c.cookieAllocator.Request(category).Raw()).
-			Done()}
+			Done(),
+			// Output the reply packet to the uplink interface if the destination is another Node's IP.
```

Comment:
Just for my understanding: if this is not a reply but a request from pod-to-node, it will go to windows host for routing and SNAT?